test/attr/dynamicReplacement_library_evolution.swift

```diff
@@ -1,6 +1,8 @@
 // RUN: %empty-directory(%t)
 // RUN: %target-typecheck-verify-swift -swift-version 5 -enable-library-evolution  -I %t
 
+// REQUIRES: objc_interop
```

Comment:
If you're not using any Objective-C imports, `-enable-objc-interop` in the RUN line is preferred.